src/components/MusicList.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div v-if="loading" id="music-list-loading">
+    <md-progress-spinner md-mode="indeterminate"></md-progress-spinner>
+    <p>{{ loadingMessage }}</p>
+  </div>
+  <div v-else id="music-list-loaded">
+    <h2 style="margin-bottom: 2rem;">
+      Music List
+      <button class="md-button"
+              style="float: right; margin-top: 0;"
+              v-on:click="addMusicDialogActive = true">
+        <md-icon>add_box</md-icon> &nbsp; <span>Add Music</span>
+      </button>
+    </h2>
+    <div v-if="musics.length" id="music-list-data">
+      <music-info v-for="(music, index) in musics"
+                  v-bind:key="music.id"
+                  v-bind:index="index"
+                  v-bind:music="music"
+                  v-on:delete="deleteMusic"
+                  v-on:play="playMusic"
```

Comment:
Curious: has everyone agreed on this delete/play/restore button setting?

If yes:
Great, it's good as it is. Though it would be better if the discussion can be documented in the project doc.
If not:
Please do so :)